virttest/qemu_vm.py

```diff
@@ -1337,6 +1337,10 @@ def sort_key(dev):
             for pcic in params.objects("pci_controllers"):
                 dev = devices.pcic_by_params(pcic, params.object_params(pcic))
                 pcics.append(dev)
+                # To avoid SeaBIOS out of IO issues always disable
+                # "io-reserve" on "pcie-root-port"
```

Comment:
Does OVMF need same configuration too? @jingzhao84 Could you help confirm? Thanks.

Reply:
For now I did not. How about setting it only when `io-reserve` is not defined?

Reply:
(I mean I considered adding it as a configuration, but this seemed more-like default than configuration so I think it should be in code. But I can add a configuration param to be able to enable it. Would that work for you?)

Reply:
it's ok, thanks